src/renderer/atlas/AtlasEngine.cpp

```diff
@@ -303,8 +302,8 @@ try
                 return blob;
             };
 
-            const auto vs = compile(SHADER_SOURCE_DIRECTORY "shader_vs.hlsl", "vs_4_1");
-            const auto ps = compile(SHADER_SOURCE_DIRECTORY "shader_ps.hlsl", "ps_4_1");
+            const auto vs = compile(_sr.sourceDirectory / L"shader_vs.hlsl", "vs_4_1");

```

Comment:
if we're only setting `_sr.sourceDirectory` when `ifndef NDEBUG`, then what happens when we're in Release?

Reply:
This section of code is also guarded by a `ifndef NDEBUG`.

Reply:
ah okay I clearly missed that